js/client.js

```diff
@@ -2217,19 +2272,71 @@
 		}
 	});
 
-	var groupDetails = this.groupDetails = {
-		'#': "Room Owner (#)",
-		'~': "Administrator (~)",
-		'&': "Leader (&amp;)",
-		'@': "Moderator (@)",
-		'%': "Driver (%)",
-		'*': "Bot (*)",
-		'\u2606': "Player (\u2606)",
-		'\u2605': "Player (\u2605)",
-		'+': "Voice (+)",
-		'‽': "<span style='color:#777777'>Locked (‽)</span>",
-		'✖': "<span style='color:#777777'>Namelocked (✖)</span>",
-		'!': "<span style='color:#777777'>Muted (!)</span>"
+	var GroupDetails = window.GroupDetails = window.GroupDetails || {
```

Comment:
OMG thank you so much - I forgot completely that there was a global called Config